ui/.yarnrc

```diff
@@ -2,5 +2,5 @@
 # yarn lockfile v1
 
 
-lastUpdateCheck 1572032507422
-yarn-path ".yarn/releases/yarn-1.19.1.js"
+lastUpdateCheck 1664829239911
+yarn-path ".yarn/releases/yarn-1.22.19.js"
```

Comment:
After updating `ember-data` to `4.5` an error was thrown stating that the yarn version was too old and the build failed. I ran `yarn policies set-version 1.22.19` as per the readme to update the embedded yarn version to the latest.